docs/FAQs.md

```diff
@@ -93,7 +93,7 @@ Here we'll outline the steps required to migrate a CTA app to a GitHub Action:
    ```diff
    +pnpm run build
    +git add dist
-   npx lint-staged
```

Comment:
This *could* be removed, but I kept this change because `pnpx` is just better. It's faster and what the project will be using after this commit.